### PR TITLE
chore(meshservice): rewrite examples using meshservice

### PR DESCRIPTION
**Pull request by slonka**
closes https://github.com/kumahq/kuma-website/issues/1993

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: app/_src/policies/meshcircuitbreaker.md [hunk 1]
```diff
@@ -457,21 +457,23 @@ spec:
...
-{% policy_yaml usage %}
+{% policy_yaml usage use_meshservice=true %}
 ```yaml
 type: MeshCircuitBreaker
 name: web-to-backend-circuit-breaker
 mesh: default
 spec:
   targetRef:
-    kind: MeshService
+    kind: MeshServiceSubset
```
Comment (lobkovilya) [inline app/_src/policies/meshcircuitbreaker.md:467]: it should be `MeshSubset` right?

File: app/_src/policies/meshtcproute.md [hunk 2]
```diff
@@ -102,57 +102,24 @@ different tags and implement A/B testing or canary deployments.
...
 ```yaml
 type: MeshTCPRoute
 name: tcp-route-1
 mesh: default
 spec:
   targetRef:
-    kind: MeshService
-    name: frontend_kuma-demo_svc_8080
+    kind: MeshSubset
+    name: frontend
```
Comment (lobkovilya) [inline app/_src/policies/meshtcproute.md:113]: needs to be `tags`

File: app/_src/policies/meshcircuitbreaker.md (hunk 1 above)
Comment (slonka) [reply app/_src/policies/meshcircuitbreaker.md:467]:
```suggestion
    kind: MeshSubset
```

File: app/_src/policies/meshtcproute.md [hunk 3]
```diff
@@ -102,57 +104,24 @@ different tags and implement A/B testing or canary deployments.
...
+    namespace: kuma-demo
+    sectionName: http
```
Comment (slonka) [inline app/_src/policies/meshtcproute.md:117]:
```suggestion
    kind: MeshSubset
    name_uni: frontend
    name_kube: frontend_kuma-demo_svc_8080
    tags:
      k8s.kuma.io/namespace: kuma-demo
```

File: jekyll-kuma-plugins/lib/jekyll/kuma-plugins/liquid/tags/policyyaml.rb [hunk 4]
```diff
@@ -58,7 +58,7 @@ def initialize(tag_name, markup, options)
                   if context[:legacy_output]
                     {
                       "kind" => "MeshService",
-                      "name" => [target_ref['name'], target_ref['namespace'], target_ref['_port']].compact.join('_')
+                      "name" => [target_ref['name'], target_ref['namespace'], "svc", target_ref['_port']].compact.join('_')
```
Comment (slonka) [inline jekyll-kuma-plugins/lib/jekyll/kuma-plugins/liquid/tags/policyyaml.rb:61]: here

File: app/_src/policies/meshfaultinjection.md [hunk 5]
```diff
@@ -154,7 +154,7 @@ spec:
         http:
           - abort:
               httpStatus: 500
-              percentage: 5
+              percentage: 50
```
Comment (slonka) [inline app/_src/policies/meshfaultinjection.md:157]: reverted oops

File: app/_src/policies/applying-policies.md [hunk 6]
```diff
@@ -135,7 +135,7 @@ This might change in the future.
 
 Apply policy with `kuma.io/effect: shadow` label:
 
-{% policy_yaml example2 %}
+{% policy_yaml example2 use_meshservice=true %}
```
[the remark this answers is not in the document]
Comment (slonka) [reply app/_src/policies/applying-policies.md:138]: You're right.

File: jekyll.yml [hunk 7]
```diff
@@ -72,6 +72,7 @@ jekyll-generator-single-source:
 include:
   - installer.sh
   - robots.txt
+  -
```
Comment (Automaat) [inline jekyll.yml:75]: is it needed?

**Review by Automaat — APPROVED** [review]

**Pull request merged**